File: src/main/java/com/unosquare/admin_core/back_end/dto/EventDTO.java
package com.unosquare.admin_core.back_end.dto;

import lombok.Data;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Data
public class EventDTO {

    private int eventId;
    private LocalDate startDate;
    private LocalDate endDate;
    private int eventTypeId;
    private String eventTypeDescription;
    private int eventStatusId;
    private String eventStatusDescription;
    private EmployeeDTO employee;
    private boolean isHalfDay;
    private LocalDateTime lastModified;
    private LocalDateTime dateCreated;
    private TeamDTO Team;
    private EventMessageDTO latestMessage;
}

File: AdminCore.DTOs/Event/EventDto.cs
using AdminCore.DTOs.Employee;
using AdminCore.DTOs.EventMessage;
using System;
using System.Collections.Generic;
using AdminCore.DTOs.EventWorkflow;

namespace AdminCore.DTOs.Event
{
  public class EventDto
  {
    public int EventId { get; set; }

    public int EmployeeId { get; set; }

    public EmployeeDto Employee { get; set; }

    public DateTime DateCreated { get; set; }

    public int EventStatusId { get; set; }

    public EventStatusDto EventStatus { get; set; }

    public int EventWorkflowId { get; set; }
    public virtual EventWorkflowDto EventWorkflow { get; set; }
    
    public int EventTypeId { get; set; }

    public EventTypeDto EventType { get; set; }

    public ICollection<EventMessageDto> EventMessages { get; set; }

    public string LatestMessage { get; set; }

    public ICollection<EventDateDto> EventDates { get; set; }

    public DateTime LastModified { get; set; }
  }
}